Dockerfile

```diff
@@ -1,5 +1,5 @@
 #FROM 648273915458.dkr.ecr.us-west-2.amazonaws.com/zilliqa:7186a35
-FROM zilliqa/zilliqa:v8.1.1-isolated-2
+FROM zilliqa/zilliqa:v8.2.0rc2
```

Comment:
@chetan-zilliqa are we removing the `isolated` term?

Reply:
Background: It's not a convention. I believe something was broken for the isolated server that led us to create `v8.1.1-isolated-2` tag.
A standard tag name should work.